app/(playground)/p/[agentId]/page.tsx

```diff
@@ -47,7 +47,7 @@ export default async function AgentPlaygroundPage({
 				uploadFileToPromptNodeFlag,
 				webSearchNodeFlag,
 				debugFlag,
-				viewFlag,
+				viewFlag: true,
```

Comment:
@toyamarinyon ❓ 

Did they forget to remove it off? 
If they didn't forget to turn it off, I would like them to delete the feature flag itself, but I understand that the feature implementation is not yet complete.

Reply:
Sorry, I made a commit by mistake.
Fixed in commit 7c47322642d0028b8bc7cf59204b414b49663311